website_baidu_map_multi/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+website_baidu_map_multi
```

Comment:
one line before (same length)
Name should be Website Baidu Map (Multi location)